services/provider/server/server.go

```diff
@@ -654,13 +655,24 @@ func (s *OCSProviderServer) GetStorageClassClaimConfig(ctx context.Context, req
 // ReportStatus rpc call to check if a consumer can reach to the provider.
 func (s *OCSProviderServer) ReportStatus(ctx context.Context, req *pb.ReportStatusRequest) (*pb.ReportStatusResponse, error) {
 	// Update the status in storageConsumer CR
-
-	if _, err := semver.Parse(req.ClientPlatformVersion); err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Malformed ClientPlatformVersion: %v", err)
+	klog.Infof("received client status: %s", req)
+
+	if req.ClientOperatorVersion == "" {
+		// distinguish b/n default vs not set
+		req.ClientOperatorVersion = notAvailable
+	} else {
+		if _, err := semver.Parse(req.ClientPlatformVersion); err != nil {
+			return nil, status.Errorf(codes.InvalidArgument, "Malformed ClientPlatformVersion: %v", err)
+		}
 	}
 
-	if _, err := semver.Parse(req.ClientOperatorVersion); err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "Malformed ClientOperatorVersion: %v", err)
+	if req.ClientPlatformVersion == "" {
+		// distinguish b/n default vs not set
```

Comment:
The comment does not make a lot of sense. Do you mean: "substituting empty values with N/A"?

Reply:
same as https://github.com/red-hat-storage/ocs-operator/pull/2251#discussion_r1384534847

Reply:
done.